src/utils.js

```diff
@@ -208,6 +197,31 @@ export function getDefaultFormState(_schema, formData, definitions={}) {
   return formData || defaults;
 }
 
+export function getUiOptions(uiSchema) {
+  // get all passed options from ui:widget, ui:options, and ui:<optionName>
+  return Object.keys(uiSchema).filter(key => /^ui:/.test(key)).reduce((options, key) => {
```

Comment:
MDN says that `String.prototype.startsWith()` is IE11- incomplient. Or is that polyfilled?